autoptsclient-zephyr.py

```diff
@@ -160,6 +160,13 @@ def main():
     stack_inst.synch_init(callback_thread.set_pending_response,
                           callback_thread.clear_pending_responses)
 
+    # Setup project PIXITS
+    autoprojects.gap.set_pixits(ptses[0])
+    autoprojects.gatt.set_pixits(ptses[0])
+    autoprojects.sm.set_pixits(ptses[0])
+    autoprojects.l2cap.set_pixits(ptses[0])
+    autoprojects.mesh.set_pixits(ptses)
```

Comment:
causes an error while running with one PTS dongle:

Traceback (most recent call last):
  File "./autoptsclient-zephyr.py", line 205, in <module>
    main()
  File "./autoptsclient-zephyr.py", line 168, in main
    autoprojects.mesh.set_pixits(ptses)
  File "/home/mskamra/projects/auto-pts/ptsprojects/zephyr/mesh.py", line 57, in set_pixits
    pts2 = ptses[1]
IndexError: list index out of range

Reply:
Yeah, I've forget about it since I'm always using two dongles. Good catch :)